doc/api/n-api.md

```diff
@@ -105,6 +105,15 @@ opt-in to access those APIs:
 In this case the entire API surface, including any experimental APIs, will be
 available to the module code.
 
+### Badges
+The use of badges is recommended to indicate the minimum version of N-API required for the module. This helps to determine which Node.js major versions are supported. The following badges are available:
+
+![N--API-v1-green](https://img.shields.io/badge/N--API-v1-green.svg)
```

Comment:
Alt text should be a descriptive version of the image, both for tool-tips and for a11y.

`![Built on N-API v2][https://...]`
`![Built on Experimental N-API][https://...]`
etc